docs/settings/telemetry-settings.asciidoc

```diff
@@ -4,32 +4,30 @@
 <titleabbrev>Telemetry settings</titleabbrev>
 ++++
 
-By default, Usage Collection (also known as Telemetry) is enabled. This
-helps us learn about the {kib} features that our users are most interested in, so we
-can focus our efforts on making them even better.
+Usage Collection (also known as Telemetry) is enabled by default. This allows us to learn what our users are most interested in, so we can improve our products and services.
+
+Refer to our https://www.elastic.co/legal/product-privacy-statement[Privacy Statement] to learn more.
 
 You can control whether this data is sent from the {kib} servers, or if it should be sent
 from the user's browser, in case a firewall is blocking the connections from the server. Additionally, you can decide to completely disable this feature either in the config file or in {kib} via *Management > Kibana > Advanced Settings > Usage Data*.
 
-Refer to our https://www.elastic.co/legal/product-privacy-statement[Privacy Statement] to learn more.
-
 [float]
 [[telemetry-general-settings]]
 ==== General telemetry settings
 
-`telemetry.sendUsageFrom`::
-  Set to `'server'` to report the cluster statistics from the {kib} server.
-  If the server fails to connect to our endpoint at https://telemetry.elastic.co/, it assumes
-  it is behind a firewall and falls back to `'browser'` to send it from users' browsers
-  when they are navigating through {kib}. Defaults to `'server'`.
-
 [[telemetry-optIn]] `telemetry.optIn`::
-  Set to `true` to send cluster statistics to Elastic. Reporting your
-  cluster statistics helps us improve your user experience. Set to `false` to stop sending any telemetry data to Elastic. +
+  Set to `false` to stop sending any telemetry data to Elastic. Reporting your
+cluster statistics helps us improve your user experience. *Default: `true`.*+
```

Comment:
Moving the default value closer to the flag definition. Being at the end of the extra note in the next paragraph made it harder to find.